addon/ng2/commands/completion.js

```diff
@@ -1,9 +1,9 @@
-/* jshint node: true */
+/*eslint-disable no-console */
```

Comment:
global console is different than what this error is, this is complaining about the use of it not the fact that its not defined.